content/_fr/policy/rules-of-use.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: sidenav
+title: "Règles d&#39;utilisation"
```

Comment:
also I'd remove all these HTML entities `&#39;` in this entire file and replace with single quotes? markdown can handle unicode

```suggestion
title: "Règles d'utilisation"
```